src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -611,6 +612,12 @@ private static BigQueryFileFormat getFileFormat(List<BigQueryTableFieldSchema> f
       if (DATETIME.equals(field.getType())) {
         return BigQueryFileFormat.NEWLINE_DELIMITED_JSON;
       }
+      // If the field name is not in english characters, then we will use json format
+      // We do this as the avro load job in BQ does not support non-english characters in field names for now
+      String fieldName = field.getName();
+      if (!Pattern.matches("[\\w]+", fieldName)) {
```

Comment:
create a variable for this regex

Reply:
```java
String wordRegex = "[\\w]+";
```

Reply:
create a static final variable with name something like COLUMN_NAME_REGEX